### PR TITLE
use api and fix api filter product by name product

### PR DESCRIPTION
**Pull request by ghost**
## Description
use api and fix api filter product by name product
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

File: public/frontend/fonts/glyphicons-halflings-regular.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
+<title>404 Not Found</title>
```
Comment (at-tuanphan) [inline public/frontend/fonts/glyphicons-halflings-regular.html]: trang 404 sao lại để trong thư muc font em?

**Review by ghost — COMMENTED** [review]
please check my comment

File: public/frontend/fonts/glyphicons-halflings-regular.html (hunk 1 above)
Comment (at-thaitran) [reply public/frontend/fonts/glyphicons-halflings-regular.html]: file này của templete em bỏ vô cho js khỏi bị lỗi thôi anh

**Comment by at-sonle** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**